src/toxic.h

```diff
@@ -1,7 +1,7 @@
 /*  toxic.h
  *
  *
- *  Copyright (C) 2014 Toxic All Rights Reserved.
+ *  Copyright (C) 2023 Toxic All Rights Reserved.
```

Comment:
2014-2024

Reply:
I've been working on this since last year :^)